api-contract/src/main/java/uk/gov/hmcts/payment/api/contract/CasePaymentOrdersDto.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.payment.api.contract;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Builder
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+public class CasePaymentOrdersDto {
+    List<CasePaymentOrderDto> content;
```

Comment:
is there any specific reason we are not using private access modifiers and going with default ?

Reply:
Updated